AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py

```diff
@@ -3,104 +3,367 @@
-        from .tests.collider import Collider_PxMeshConvexMeshCollides as test_module
-        self._run_test(request, workspace, editor, test_module)
     
-    @revert_physics_config    
-    def test_ShapeCollider_CylinderShapeCollides(self, request, workspace, editor, launcher_platform):
+    class ShapeCollider_CylinderShapeCollides(EditorSharedTest):
         from .tests.shape_collider import ShapeCollider_CylinderShapeCollides as test_module
-        self._run_test(request, workspace, editor, test_module)
-
-    @revert_physics_config
-    def test_C15425929_Undo_Redo(self, request, workspace, editor, launcher_platform):
-        from .tests import Physics_UndoRedoWorksOnEntityWithPhysComponents as test_module
-        self._run_test(request, workspace, editor, test_module, enable_prefab_system=False)
 
     @pytest.mark.GROUP_tick
     @pytest.mark.xfail(reason="Test still under development.")
-    def test_Tick_InterpolatedRigidBodyMotionIsSmooth(self, request, workspace, editor, launcher_platform):
+    class Tick_InterpolatedRigidBodyMotionIsSmooth(EditorSharedTest):
         from .tests.tick import Tick_InterpolatedRigidBodyMotionIsSmooth as test_module
-        self._run_test(request, workspace, editor, test_module)
 
     @pytest.mark.GROUP_tick
     @pytest.mark.xfail(reason="Test still under development.")
-    def test_Tick_CharacterGameplayComponentMotionIsSmooth(self, request, workspace, editor, launcher_platform):
+    class Tick_CharacterGameplayComponentMotionIsSmooth(EditorSharedTest):
         from .tests.tick import Tick_CharacterGameplayComponentMotionIsSmooth as test_module
-        self._run_test(request, workspace, editor, test_module)
+
+    class Collider_BoxShapeEditing(EditorSharedTest):
+        from .tests.collider import Collider_BoxShapeEditing as test_module
+
+    class Collider_SphereShapeEditing(EditorSharedTest):
+        from .tests.collider import Collider_SphereShapeEditing as test_module
+
+    class Collider_CapsuleShapeEditing(EditorSharedTest):
+        from .tests.collider import Collider_CapsuleShapeEditing as test_module
+
+    class Collider_CheckDefaultShapeSettingIsPxMesh(EditorSharedTest):
+        from .tests.collider import Collider_CheckDefaultShapeSettingIsPxMesh as test_module
+
+    class Collider_MultipleSurfaceSlots(EditorSharedTest):
+        from .tests.collider import Collider_MultipleSurfaceSlots as test_module
+
+    class Collider_PxMeshAutoAssignedWhenAddingRenderMeshComponent(EditorSharedTest):
+        from .tests.collider import Collider_PxMeshAutoAssignedWhenAddingRenderMeshComponent as test_module
+        
+    @pytest.mark.xfail(reason="AssertionError: Couldn't find Asset with path: Objects/SphereBot/r0-b_body.azmodel")
+    class Collider_PxMeshAutoAssignedWhenModifyingRenderMeshComponent(EditorSharedTest):
+        from .tests.collider import Collider_PxMeshAutoAssignedWhenModifyingRenderMeshComponent as test_module
```

Comment:
This test had the wrong xfail reason. It's failing because it's trying to use an asset the doesn't exist in AutomatedTesting.

This will have to be addressed at later.